install/helm/agones/templates/crds/_gameserverspecschema.yaml

```diff
@@ -153,6 +153,42 @@ properties:
             type: integer
             title: The initial player capacity of this Game Server
             minimum: 0
+{{- if .featureCountsAndLists }}
+      counters:
+        type: array
+        title: Array of player, room, session, etc. counters
+        nullable: true
+        maxItems: 1000
```

Comment:
Nice touch. I can't imaging anyone will need this many, but nice to have a top limit.

Reply:
Hopefully no one will hit the limit. It's recommended for CEL validation which we may do in the future https://kubernetes.io/blog/2022/09/23/crd-validation-rules-beta/#writing-validation-rules-in-openapiv3-schemas

Reply:
Hopefully no one will hit the limit :) Putting in a limit was recommended for if we do CEL validation in the future.